gm4_lightning_in_a_bottle/data/gm4_lightning_in_a_bottle/functions/potion_tracker/check_splash.mcfunction

```diff
@@ -0,0 +1,6 @@
+# @s = none
+# scheduled from potion_tracker/spawn_tracker
+
+execute at @e[type=marker,tag=gm4_liab_potion_tracker] unless entity @e[type=#gm4_lightning_in_a_bottle:potion_tracker,tag=gm4_liab_potion_tracker,distance=.001..1,limit=1] run function gm4_lightning_in_a_bottle:potion_tracker/potion_landed
+
+execute if entity @e[type=marker,tag=gm4_liab_potion_tracker] run schedule function gm4_lightning_in_a_bottle:potion_tracker/check_splash 1t
```

Comment:
This potentially causes issues with high potion velocities, at which point this causes massive client and server side performance drops as the potion will think it has landed every tick until it actually hits something.